compare_to_torch.py

```diff
@@ -0,0 +1,65 @@
+import torch
```

Comment:
I'm not sure we want to keep this script in the main repo, but if we do, perhaps we can move it in the `bin` folder?

Reply:
I really wasn't sure what to do with this. Its really helpful to me when developing, but I also don't think the main repo is the right place. `bin` also sounds good to me, or I can also remove it from the repo.

Reply:
I use this for development and I kinda like it that I can change the range for example of the values.

Reply:
I think code like that tends to bit rot. Maybe it's better to turn this into actual tests that are guarded with `has_torch`? We already have a bunch of tests that use PyTorch and install PyTorch in CI. That would have the benefit that these tests are always run in CI as well.

Reply:
Great idea, let's do that :-)

Reply:
Thanks for the suggest :-). I've pushed the test in `thinc/tests/backends/test_ops.py` with `has_torch`